tests/unit/services/ajax-test.js

```diff
@@ -134,6 +134,49 @@ test("request() promise label is correct", function(assert) {
   assert.equal(postPromise._label, 'ember-ajax: POST to /posts');
 });
 
+test("post() promise label is correct", function(assert) {
+  service = Service.create();
+  const url = '/posts';
```

Comment:
As I understand it, `const` means "constant reference", not "constant value", @stefanpenner...is this wrong?